framework/src/org/apache/cordova/CordovaWebView.java

```diff
@@ -675,13 +675,6 @@ public boolean dispatchKeyEvent(KeyEvent event)
                         return true;
                     }
                     // If not, then invoke default behavior
```

Comment:
This comment does not seem to be referring to anything anymore. I guess it can be removed, or moved elsewhere?